src/content/protocols/compound-v3-arbitrum.md

```diff
@@ -0,0 +1,219 @@
+
+> Upgradeability score: High
+
+## Autonomy
+
+The compound-v3 protocol relies on a Chainlink oracle feed to price collateral and base assets in the system. The protocol does not validate asset prices returned by Chainlink or offer a fallback oracle mechanism. The replacement of a stale or untrusted oracle feed requires a Compound governance vote with a delay (see [Exit Window](#exit-window)).
+
+The Chainlink oracle system itself is upgradeable without decentralized ownership over those permissions. This dependency thus introduces centralization risk in the Compound-v3 protocol.
+
+> Autonomy score: High
+
+## Exit Window
+
+Permissions, including protocol upgrades, are controlled by an onchain governance system on Ethereum mainnet. `COMP` holders are able to create new proposals (requires 25,000 `COMP`) and vote on proposals (at least 400,000 votes are required for a valid proposal). A minimum voting period of 3 days is enforced as well as a delay of 3 days for the implementation of successful proposals.
+
+While this does not meet the 7-day exit window requirement, malicious or unintended proposals can be intercepted by the `ProposalGuardian` multisig account.
+
+However, the `ProposalGuardian` multisig accounts does not meet the Security Council requirements ([see below](#security-council)).
+
+> Exit Window score: High
+
+## Accessibility
+
+The frontend of Compound V3 is open source. Instructions to deploy it locally or deploy it
+on IPFS are available [here](https://github.com/compound-finance/palisade). However, the frontend depends on a separate backend that is accessed through [v3-api.compound.finance/](https://v3-api.compound.finance/). Therefore, self-hosting does not help if the backend is down.
+Nonetheless, compound is supported on third-party apps like DeFiSaver. These apps build an acceptable backup solution in case of failure of the official frontend and backend.
+
+> Accessibility score: Low
+
+## Conclusion
+
+The Compound-v3 Ethereum mainnet protocol achieves High centralization risk scores for its _Upgradeability_, _Autonomy_ and _Exit Window_ dimensions. It thus ranks Stage 0.
+
+The protocol could reach Stage 1 by 1) adopting a _Security Council_ setup for the `ProposalGuardian` multisig accounts, and 2) implementing validity checks and a fallback mechanism around the Chainlink oracle (or Chainlink adopting a _Security Council_ setup for its own multisig account).
+
+> Overall score: Stage 0
+
+# Technical Analysis
+
+Below is an overview of the contracts from the Compound V3 protocol.
+
+![Overview of the compound protocol](./diagrams/compound-v3-arbitrum-overview.png)
+
+⚠️ During our analysis, we noticed many of the contract addresses listed in the [official documentation](https://docs.compound.finance/) are out of date. This is most likely explained by the high frequency of updates to the implementation contracts. The list below was last updated on the 20th of February 2025.
```

Comment:
move statement (and line break) below "Contracts" section (same as in Mainnet review)